modules/candidate_parameters/ajax/formHandler.php

```diff
@@ -149,6 +149,44 @@ function editProbandInfoFields($db, $user)
                     ];
 
     $db->update('candidate', $updateValues, ['CandID' => $candID]);
+    foreach (array_keys($_POST) as $field) {
+        if (!empty($_POST[$field])) {
+            if (substr($field, 0, 4) === 'PTID') {
+                $ptid = substr($field, 4);
+
+                $updateValues = [
+                                 'ParameterTypeID' => $ptid,
+                                 'CandID'          => $candID,
+                                 'Value'           => $_POST[$field],
+                                 'InsertTime'      => time(),
+                                ];
+
+                $result = $db->pselectOne(
+                    'SELECT * from parameter_candidate 
```

Comment:
SELECT * should not be used. If you just want to confirm that no entry exist for from parameter, you should select the primary key of that table (i.e.: SELECT CandID FROM ...)